examples/2d/compute_speed.py

```diff
@@ -25,13 +25,39 @@
 # Certain backends are also GPU-only, we we want to detect that before running
 # the benchmark.
 
-import kymatio.scattering2d.backend as backend
+backends = []
+
+try:
+    if torch.cuda.is_available():
+        from skcuda import cublas
+        import cupy
+        from kymatio.scattering2d.backend import torch_skcuda_backend
+        backends.append(torch_skcuda_backend)
+except:
+    pass
+
+try:
+    from kymatio.scattering2d.backend import torch_backend
+    backends.append(torch_backend)
+except:
+    pass
+
+
+###############################################################################
+# Determine which devices (CPU or GPU) that are supported by the current
+# backend.
+
+if torch.cuda.is_available():
```

Comment:
this benchmark either needs to be renamed to `compute_speed_torch` or make the torch/tf import optional using try

Reply:
benchmarks must be backend dependant